### PR TITLE
feat: add autogenerated documentation for entities

### PR DESCRIPTION
**Pull request by yrobla**

### CODE REVIEW

**Comment by yrobla** [comment]
image

**Comment by yrobla** [comment]
It actually misses adding comments to the classes and methods we want to document, so i will be doing that

**Comment by yrobla** [comment]
image

File: site/docusaurus.config.js [hunk 1]
```diff
@@ -21,20 +21,18 @@ const theButton = require('./src/rehype/thebutton');
     url: config.url,
     baseUrl: '/',
     trailingSlash: true,
-    onBrokenLinks: 'throw',
-    onBrokenMarkdownLinks: 'throw',
+    onBrokenLinks: 'warn',
+    onBrokenMarkdownLinks: 'warn',
```
Comment (aguillenv) [inline site/docusaurus.config.js:25]: do we want or need this for any reason?

File: site/docusaurus.config.js [hunk 2]
```diff
@@ -91,17 +90,17 @@ const theButton = require('./src/rehype/thebutton');
               label: 'Dashboard',
             },
             {
-              to: 'docs',
+              href: '/docs',
```
Comment (aguillenv) [inline site/docusaurus.config.js:93]: this change is because the plugin does not understand the `to:` property?

File: site/typedoc.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "entryPoints": ["../src/modules/*" ],
+  "entryPointStrategy": "expand",
+  "cleanOutputDir": true,
+  "excludeExternals": true,
+  "name": "IaSQL documentation",
+  "categorizeByGroup": false,
+  "disableSources": true,
+  "readme": "./sql_reference.md",
```
Comment (aguillenv) [inline site/typedoc.json:9]: do we have this file?

File: site/docusaurus.config.js (hunk 1 above)
Comment (mtp1376) [reply site/docusaurus.config.js:25]: +1

**Review by mtp1376 — APPROVED** [review]

File: site/typedoc.json (hunk 3 above)
Comment (yrobla) [reply site/typedoc.json]: this is actually not used. The plugin just checks if the readme is set to some value or to "none"

**Review by aguillenv — APPROVED** [review]

**Comment by dfellis** [comment]
> image

This is great! But could we group the tables by the module? The current alphabetically-sorted structure is going to be overwhelming, I think, and without the module mention will be confusing to users when they don't have these tables automatically in their database.

**Review by dfellis — APPROVED** [review]

**Review by depombo — APPROVED** [review]

**Pull request merged**